babel.config.js

```diff
@@ -1,32 +0,0 @@
-const defines = require('./babel-defines')
-
-function replacementPlugin(env) {
-  return ['babel-plugin-transform-replace-expressions', {replace: defines[env]}]
-}
-
-const sharedPlugins = ['macros', 'preval', 'add-react-displayname', 'babel-plugin-styled-components', '@babel/plugin-proposal-nullish-coalescing-operator']
```

Comment:
TODO: Review these plugins and supply any missing functionality.

- [`typescript-plugin-styled-components`](https://styled-components.com/docs/tooling#typescript-plugin) was installed to replace `babel-plugin-styled-components`
- [Nullish Coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing) is built into TypeScript